storagenode/storagenodedb/database.go

```diff
@@ -56,27 +101,38 @@ func New(log *zap.Logger, config Config) (*DB, error) {
 	}
 	pieces := filestore.New(piecesDir)
 
-	infodb, err := newInfo(config.Info2)
-	if err != nil {
-		return nil, err
-	}
-
 	dbs, err := boltdb.NewShared(config.Kademlia, kademlia.KademliaBucket, kademlia.NodeBucket, kademlia.AntechamberBucket)
 	if err != nil {
 		return nil, err
 	}
 
-	return &DB{
+	db := &DB{
 		log: log,
 
 		pieces: pieces,
 
-		info: infodb,
-
 		kdb: dbs[0],
 		ndb: dbs[1],
 		adb: dbs[2],
```

Comment:
I know this wasn't your change, but it make sme uncomfortable that we're using these indexes without checking the length of `dbs`. I recognize that if we get a nil error in the `boltdb.NewShared` then we're guaranteed to have these indexes, but that's not guaranteed in the future. Maybe we want a panic if someone makes that change accidentally? But I wouldn't mind a length check here.